.github/actions/README.md

```diff
@@ -0,0 +1,18 @@
+# Actions
+
+This directory contains code for GitHub Actions that are used in this repo's workflow.
+
+The actions are built with yarn like the rest of this repo.
+
+To build / update the actions, run this inside the action's directory:
+
+```
+yarn
```

Comment:
[q] Is this line neded?

Reply:
Good question. Currently if you don't run it the dependencies are not installed. Is there a better way in yarn to automat this than to add that call to the `run all` script?

Reply:
yes